source/extensions/common/wasm/wasm.h

```diff
@@ -1072,6 +1078,12 @@ struct SaveRestoreContext {
   uint32_t saved_effective_context_id_;
 };
 
+inline PluginDirection
+pluginDirectionFromTrafficDirection(envoy::api::v2::core::TrafficDirection direction) {
+  return direction == envoy::api::v2::core::TrafficDirection::INBOUND ? PluginDirection::Inbound
```

Comment:
We should default unspecified as inbound in istio context (since older control planes don't set the field).

Reply:
That would mean changing the value from UNSPECIFIED to INBOUND.  Rather I think we should just document in the istio plugin docs that UNSPECIFIED should be interpreted as INBOUND where applicable.